ui/app/AppLayouts/Communities/popups/CommunityMemberMessagesPopup.qml

```diff
@@ -0,0 +1,141 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+import QtQml.Models 2.14
+import QtGraphicalEffects 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Popups 0.1
+import StatusQ.Popups.Dialog 0.1
+
+import utils 1.0
+import shared.views.chat 1.0
+
+StatusDialog {
+    id: root
+
+    property var store
+    property var chatCommunitySectionModule
+    property var memberMessagesModel: chatCommunitySectionModule.memberMessagesModel
+    property string memberPubKey: ""
+    property string displayName: ""
+
+
+    width: 800
+    height: 428
```

Comment:
Don't hardcode the height, make the contents scrollable using `StatusScrollView`

Reply:
sorry, maybe I missed something, but it is already scrollable (see on video admin messages scrolling)
This way I set up fixed width and height of the popup

Reply:
The height should be dynamic, based on the window height (the default `StatusDialog` behavior)

Reply:
If I remove height, I'm getting a corrupted UI (it seems height is 0)

![image](https://github.com/status-im/status-desktop/assets/117639195/38201e5c-6c48-43af-bd3d-929fcdb5b2f7)

Do we have some similar popup dialog? In pinned messages, the height is hardcoded too

Reply:
You need to use the scroll view this way:
```qml
    StatusScrollView {
        anchors.fill: parent
        contentWidth: availableWidth

        ColumnLayout {
            anchors.left: parent.left
            anchors.leftMargin: 40
            anchors.right: parent.right
            anchors.rightMargin: 40
            ...
```

i.e. no `contentItem`

Reply:
Did you push the change, I don't see StatusScrollView being used there